price-feed-wrapper/price-feed-wrapper.sol

```diff
@@ -0,0 +1,81 @@
+import "substrate";
+import {IPriceOracleGetter} from "./interfaces/IPriceOracleGetter.sol";
+
+/**
+ * @title PriceFeedWrapper
+ * @notice Price oracle that uses the native chain via chain extension. It stores _asset, _blockchain and _symbol for use by function getAssetPrice() which is called by Nabla. 
+ * @dev This contract can be used with the Nabla's Router contract
+ */
+contract PriceFeedWrapper is IPriceOracleGetter {
```

Comment:
I think I would rename this. Maybe call it `PriceOracleWrapper` since we are providing an implementation for `IPriceOracleGetter`?

Reply:
I'd also rename the file and directory accordingly then

Reply:
Makes sense, thanks.